sources/packages/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -214,13 +207,19 @@ export class ApplicationStudentsController extends BaseController {
       userToken.userId,
     );
 
-    await this.validateOverlappingDatesAndPIR(
+    this.applicationService.setBypassApplicationSubmitValidations(
+      this.config.bypassApplicationSubmitValidations,
```

Comment:
I would recommend moving this inside the constructor and setting the variable as read-only since it will not change.